AUTHORS

```diff
@@ -3,7 +3,12 @@
 # This does not necessarily list everyone who has contributed code, since in
 # some cases, their employer may be the copyright holder.  To see the full list
 # of contributors, see the revision history in source control.
+
+# Companies
```

Comment:
If this was a separate pull request, it should be more visible.

Reply:
Pushed into a separate pull request.